recipes/raysect/recipe.yaml

```diff
@@ -0,0 +1,62 @@
+context:
+  name: raysect
+  version: "0.8.1"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://github.com/${{ name|lower }}/source/archive/refs/tags/v${{ version }}.tar.gz
+  sha256: 6b47f06cf58e6bf021606aa3a7b42f0a200f63d8695e01a6274941dedf3685bb
+
+build:
+  script: ${{ PYTHON }} -m pip install --no-build-isolation --no-deps -vv .
+  number: 0
+  skip:
+    - win
+
+requirements:
+  build:
+    - if: unix
+      then:
+        - ${{ compiler("c") }}
+    - if: win
+      then:
+        - ${{ compiler("clang") }}
+    - ${{ stdlib("c") }}
+  host:
+    - python
+    - pip
+    - setuptools
+    - cython <3.0
+    - numpy <2.0
+  run:
+    - python
+    - numpy <2.0
```

Comment:
If numpy is in host, the run dependency comes through run_exports 
```suggestion
```

Reply:
Did you mention `pin_compatible("numpy")`?

Reply:
Sorry I am confused.
what you mentined is that we should put same `numpy` in the run dependency in order to appropriately link against `numpy` in the host environment, right?
So, should I replace `numpy <2.0` into `${{ pin_subpackage("numpy", exact=True) }}` or the following expression?

```yaml
run_exports:
    - ${{ pin_subpackage("numpy", exact=True) }}
```

If I tried to opt for the latter, the following error was occurred:
`Error:   × Missing output: numpy (used in pin_subpackage)`

Reply:
No, I want you to exactly do what I suggested, delete `numpy` completely from the explicitly listed run requirements.

Reply:
Ah, so if `numpy` is specified in host deps, it is automatically tied to run requirement through `run_export`.
I got it.